sfm_pc/utils.py

```diff
@@ -429,12 +437,25 @@ def format_facets(facet_dict):
 
     return out
 
-def get_command_edges(org_id, when=None):
-    edge_list = get_org_hierarchy_by_id(org_id, when=when)
+def get_chart_edges(org_id, when=None):
```

Comment:
Do you mind if we keep the name `get_command_edges` for this function? I actually wound up re-using it in some code over the weekend, out of context of a chart! (We do some new data validation stuff in the forms to make sure that command hierarchies aren't recursive.)

Reply:
Oh, of course! I'll change it back, and then, rename my other new function for consistency.